jupyterhub/jupyterhub/overlays/jupyterhub-idle-culler/deploymentconfig.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: jupyterhub-idle-culler
+spec:
+  selector:
+    app: jupyterhub-idle-culler
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: jupyterhub-idle-culler
+    spec:
+      containers:
+        - name: jupyterhub-idle-culler
+          image: quay.io/odh-jupyterhub/jupyterhub-idle-culler:latest
```

Comment:
We will want to make a release in the idle culler image repo and point to that released image version here

Reply:
can we not just add the JH culler package to the [jupyterhub image pipfile](https://github.com/opendatahub-io/jupyterhub-odh/blob/master/Pipfile)? This would mean one less image for us to maintain and this would also mean that users would have the option to use it as a JH service if they wanted.